lib/analyzers/two_fer/analyze.rb

```diff
@@ -1,19 +1,18 @@
 module TwoFer
-
   MESSAGES = {
-    no_module:               "ruby.general.no_target_module",
-    no_method:               "ruby.general.no_target_method",
-    incorrect_indentation:   "ruby.general.incorrect_indentation",
-    explicit_return:         "ruby.general.explicit_return",         # "The last line automatically gets returned"
-    splat_args:              "ruby.two-fer.splat_args",              # "Rather than using *%s, how about actually setting a parameter called 'name'?"
-    missing_default_param:   "ruby.two-fer.missing_default_param",   # "There is no correct default param - the tests will fail"
+    no_module: "ruby.general.no_target_module",
+    no_method: "ruby.general.no_target_method",
+    incorrect_indentation: "ruby.general.incorrect_indentation",
+    explicit_return: "ruby.general.explicit_return", # "The last line automatically gets returned"
+    splat_args: "ruby.two-fer.splat_args", # "Rather than using *%s, how about actually setting a parameter called 'name'?"
```

Comment:
Not a request to change it in this PR but "how'bout actually" seems a bit less ideal than "what if we set a argument called "name"?